cgi-bin/Apache/BML.pm

```diff
@@ -1916,7 +1916,7 @@ sub get_path_info
 sub get_remote_ip
 {
     my $apache_r = BML::get_request();
```

Comment:
Wouldn't it be easier to update callers to use DW::Request?

If not, then I have no objections.